drivers/analog/comp.c

```diff
@@ -91,35 +91,12 @@ static const struct comp_callback_s g_comp_callback =
 static void comp_pollnotify(FAR struct comp_dev_s *dev,
                             pollevent_t eventset)
 {
-  int i;
-
   if (eventset & POLLERR)
     {
       eventset &= ~(POLLOUT | POLLIN);
```

Comment:
don't need, handled by poll_notify now. let's remove comp_pollnotify and call poll_notify directly

Reply:
Done